mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -494,8 +496,17 @@ def test_spectral_connectivity_epochs_multivariate(method):
     if method in ['mic', 'mim']:
         con = spectral_connectivity_epochs(
             data, method=method, mode=mode, indices=None, sfreq=sfreq)
-        assert (np.array(con.indices).tolist() ==
-                [[[0, 1, 2, 3]], [[0, 1, 2, 3]]])
```

Comment:
Why do we change this? 

Is this something that changes the behavior of `indices = None`?

Reply:
In the docstring, the behavior for `None` states:

```
If None, connections between all channels
        are computed, unless a Granger causality method is called, in which
        case an error is raised.
```

Reply:
I realised that if you pass `indices=None` for bivariate methods, the `indices` attribute of the connectivity object will be `None`, so this was my attempt at making the behaviour more consistent.

If you think it is better to return the explicit indices, I can switch it back.

Reply:
Okay sure that's fine. Let's ere on the side of keeping behavior consistent w/ whatever it used to be to limit backwards-incompatible changes.

So this change is fine then w/ me.